src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ContentDialogTests/ContentDialog_Closing.xaml.cs

```diff
@@ -58,14 +58,14 @@ private void DeferredDialog_Click(object sender, RoutedEventArgs args)
 			var defer1Button = new _Button { Name = "Complete1Button", Content = "Complete 1" };
 			defer1Button.Click += (o, e) =>
 			{
-				deferral1.Complete();
 				ResultTextBlock.Text = "First complete called";
+				deferral1.Complete();
```

Comment:
Why was this change needed?

Reply:
I observed that when all deferral's were being completed in the Click handler, the execution of the click handler was stopped. Probably due to the Button going out of scope when the dialog got dismissed. This caused the TextBlock not being updated and the UI test failing. I wasn't completely sure what caused it, but I felt like the change was minor enough.

Reply:
> Can you add a new UI test that verifies the sample you added?
I could try to do that. I'm not very comfortable with the test framework yet though.